compiler/compile/OMRCompilation.cpp

```diff
@@ -2739,3 +2742,29 @@ bool OMR::Compilation::incompleteOptimizerSupportForReadWriteBarriers()
    {
    return false;
    }
+
+int32_t OMR::Compilation::getNumReservedIPICTrampolines()
+   {
+#ifdef TR_TARGET_X86
+   return self()->cg()->getNumReservedIPICTrampolines();
+#else
+   return 0;
+#endif
+   }
+
+void OMR::Compilation::setNumReservedIPICTrampolines(int32_t n)
+   {
+#ifdef TR_TARGET_X86
+   return self()->cg()->setNumReservedIPICTrampolines(n);
```

Comment:
I guess this technically works, but given that the function returns void, would it make more sense to skip the `return`?

```suggestion
   self()->cg()->setNumReservedIPICTrampolines(n);
```